core/livepeernode.go

```diff
@@ -259,6 +259,12 @@ func (n *LivepeerNode) transcodeAndBroadcastSeg(seg *stream.HLSSegment, sig []by
 		return // TODO return error?
 	}
 
+	// Ensure length matches expectations. 4 second + 25% wiggle factor, 60fps
+	err := ffmpeg.CheckMediaLen(fname, 4*1.25*1000, 60*4*1.25)
+	if err != nil {
+		glog.Errorf("Media length check failed: %v", err)
```

Comment:
Yeah -- right now we don't distinguish between invalid duration or invalid packets from the error-handling code. I looked into doing that, but it doesn't quite fit as neatly into our current error-handling scheme, so I punted on the problem for now, especially since we don't have a way to actually act on the information.

Can add that in if you feel it's important to have sooner rather than later.

Reply:
I don't think it is critical to add that now! I was just thinking about the utility of propagating the error message when we actually do have a way to act on the information